src/manifest.ts

```diff
@@ -504,7 +504,7 @@ export class Manifest {
     if (fromSha === undefined) {
       fromSha = (await this.getConfigJson())['bootstrap-sha'];
     }
-    return this.gh.commitsSinceSha(fromSha);
+    return this.gh.commitsSinceShaRest(fromSha);
```

Comment:
Is there any food reason for us to keep `commitsSinceSha`?

Perhaps just open a tracking ticket to get rid of it? (I understand not wanting to do it as part of this work).

Reply:
created https://github.com/googleapis/release-please/issues/830

Reply:
Thank you 👏